<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fltLogEntryForm</class>
 <widget class="QWidget" name="fltLogEntryForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>582</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>New Flight Entry</string>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <item>
        <widget class="QDateEdit" name="flt_dateEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>0</height>
          </size>
         </property>
         <property name="wrapping">
          <bool>false</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="accelerated">
          <bool>false</bool>
         </property>
         <property name="correctionMode">
          <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
         </property>
         <property name="dateTime">
          <datetime>
           <hour>0</hour>
           <minute>0</minute>
           <second>0</second>
           <year>2000</year>
           <month>1</month>
           <day>1</day>
          </datetime>
         </property>
         <property name="time">
          <time>
           <hour>0</hour>
           <minute>0</minute>
           <second>0</second>
          </time>
         </property>
         <property name="displayFormat">
          <string>dd/MM/yyyy</string>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
         <property name="date">
          <date>
           <year>2000</year>
           <month>1</month>
           <day>1</day>
          </date>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>ttbd</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <item>
        <widget class="QGroupBox" name="dep_groupBox">
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="title">
          <string>Departure</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout">
          <item row="0" column="0">
           <layout class="QVBoxLayout">
            <item>
             <widget class="QTimeEdit" name="dep_timeEdit">
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::UpDownArrows</enum>
              </property>
              <property name="correctionMode">
               <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
              </property>
              <property name="minimumTime">
               <time>
                <hour>0</hour>
                <minute>0</minute>
                <second>0</second>
               </time>
              </property>
              <property name="displayFormat">
               <string>hh:mm</string>
              </property>
              <property name="calendarPopup">
               <bool>false</bool>
              </property>
              <property name="time">
               <time>
                <hour>22</hour>
                <minute>59</minute>
                <second>59</second>
               </time>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="dep_comboBox">
              <property name="editable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="arr_groupBox">
         <property name="title">
          <string>Arrival</string>
         </property>
         <layout class="QGridLayout">
          <item row="0" column="0">
           <layout class="QVBoxLayout">
            <item>
             <widget class="QTimeEdit" name="arr_timeEdit">
              <property name="displayFormat">
               <string>hh:mm</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="arr_comboBox">
              <property name="editable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="ac_groupBox">
         <property name="title">
          <string>Aircraft / PIC</string>
         </property>
         <layout class="QGridLayout">
          <item row="0" column="0">
           <layout class="QVBoxLayout">
            <item>
             <widget class="QComboBox" name="ac_comboBox">
              <property name="editable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="pic_comboBox">
              <property name="editable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="fltRules_groupBox">
         <property name="title">
          <string>Flight Rules</string>
         </property>
         <layout class="QGridLayout">
          <item row="0" column="0">
           <layout class="QVBoxLayout">
            <item>
             <widget class="QComboBox" name="fr_comboBox">
              <item>
               <property name="text">
                <string>a</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>as</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>at</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QTimeEdit" name="fr_timeEdit">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="displayFormat">
               <string>hh:mm</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="ldg_groupBox">
         <property name="title">
          <string>Landing</string>
         </property>
         <layout class="QGridLayout">
          <item row="0" column="0">
           <layout class="QGridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="day_label">
              <property name="text">
               <string>&amp;Day</string>
              </property>
              <property name="buddy">
               <cstring>day_spinBox</cstring>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="day_spinBox">
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::PlusMinus</enum>
              </property>
              <property name="maximum">
               <number>9</number>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="nit_label">
              <property name="text">
               <string>&amp;Night</string>
              </property>
              <property name="buddy">
               <cstring>nit_spinBox</cstring>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="nit_spinBox">
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::PlusMinus</enum>
              </property>
              <property name="maximum">
               <number>9</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <item>
        <widget class="QGroupBox" name="rem_groupBox">
         <property name="title">
          <string>Remarks</string>
         </property>
         <layout class="QGridLayout">
          <item row="0" column="0">
           <widget class="QTextEdit" name="remarks_textEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout">
         <item>
          <widget class="QGroupBox" name="log_groupBox">
           <property name="title">
            <string>Pilot Function</string>
           </property>
           <layout class="QGridLayout">
            <item row="0" column="0">
             <layout class="QVBoxLayout">
              <item>
               <widget class="QComboBox" name="func_comboBox"/>
              </item>
              <item>
               <widget class="QCheckBox" name="nit_checkBox">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Auto Night</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
                <property name="tristate">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout">
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout">
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_2">
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <item>
        <widget class="QGroupBox" name="pilot_groupBox">
         <property name="title">
          <string>Aircaft Pilot Time</string>
         </property>
         <layout class="QGridLayout">
          <item row="0" column="0">
           <layout class="QGridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="sesp_label">
              <property name="text">
               <string>Single Engine</string>
              </property>
              <property name="buddy">
               <cstring>sse_time</cstring>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QTimeEdit" name="sse_time">
              <property name="frame">
               <bool>false</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
              <property name="displayFormat">
               <string>hh:mm</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="spme_label">
              <property name="text">
               <string>M&amp;ulti Engine</string>
              </property>
              <property name="buddy">
               <cstring>sme_time</cstring>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QTimeEdit" name="sme_time">
              <property name="frame">
               <bool>false</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
              <property name="displayFormat">
               <string>hh:mm</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="mcc_label">
              <property name="text">
               <string>Mu&amp;lti Pilot</string>
              </property>
              <property name="buddy">
               <cstring>mcc_timeEdit</cstring>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QTimeEdit" name="mcc_timeEdit">
              <property name="frame">
               <bool>false</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
              <property name="displayFormat">
               <string>hh:mm</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout">
         <item>
          <widget class="QGroupBox" name="tot_groupBox">
           <property name="title">
            <string>Total Time</string>
           </property>
           <layout class="QGridLayout">
            <item row="0" column="0">
             <widget class="QTimeEdit" name="tot_timeEdit">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="frame">
               <bool>false</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
              <property name="displayFormat">
               <string>hh:mm</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="ops_groupBox">
           <property name="title">
            <string>Operational Conditions</string>
           </property>
           <layout class="QGridLayout">
            <item row="0" column="0">
             <layout class="QGridLayout">
              <item row="0" column="0">
               <widget class="QLabel" name="ifr_time_label">
                <property name="text">
                 <string>IFR</string>
                </property>
                <property name="buddy">
                 <cstring>ifr_timeEdit</cstring>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLabel" name="nit_time_label">
                <property name="text">
                 <string>Ni&amp;ght</string>
                </property>
                <property name="buddy">
                 <cstring>nit_timeEdit</cstring>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QTimeEdit" name="ifr_timeEdit">
                <property name="frame">
                 <bool>false</bool>
                </property>
                <property name="buttonSymbols">
                 <enum>QAbstractSpinBox::NoButtons</enum>
                </property>
                <property name="displayFormat">
                 <string>hh:mm</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QTimeEdit" name="nit_timeEdit">
                <property name="frame">
                 <bool>false</bool>
                </property>
                <property name="buttonSymbols">
                 <enum>QAbstractSpinBox::NoButtons</enum>
                </property>
                <property name="displayFormat">
                 <string>hh:mm</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="func_groupBox">
         <property name="title">
          <string>Pilot Function Time</string>
         </property>
         <layout class="QGridLayout">
          <item row="0" column="0">
           <layout class="QGridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="pic_label">
              <property name="text">
               <string>PIC</string>
              </property>
              <property name="buddy">
               <cstring>pic_timeEdit</cstring>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="copi_label">
              <property name="text">
               <string>COPI</string>
              </property>
              <property name="buddy">
               <cstring>copi_timeEdit</cstring>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QTimeEdit" name="pic_timeEdit">
              <property name="wrapping">
               <bool>false</bool>
              </property>
              <property name="frame">
               <bool>false</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
              <property name="displayFormat">
               <string>hh:mm</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QTimeEdit" name="copi_timeEdit">
              <property name="wrapping">
               <bool>false</bool>
              </property>
              <property name="frame">
               <bool>false</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
              <property name="displayFormat">
               <string>hh:mm</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="dual_label">
              <property name="text">
               <string>DUAL</string>
              </property>
              <property name="buddy">
               <cstring>dual_timeEdit</cstring>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="instr_label">
              <property name="text">
               <string>INS&amp;TR</string>
              </property>
              <property name="buddy">
               <cstring>instr_timeEdit</cstring>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QTimeEdit" name="dual_timeEdit">
              <property name="wrapping">
               <bool>false</bool>
              </property>
              <property name="frame">
               <bool>false</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
              <property name="displayFormat">
               <string>hh:mm</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QTimeEdit" name="instr_timeEdit">
              <property name="wrapping">
               <bool>false</bool>
              </property>
              <property name="frame">
               <bool>false</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
              <property name="displayFormat">
               <string>hh:mm</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>flt_dateEdit</tabstop>
  <tabstop>dep_timeEdit</tabstop>
  <tabstop>arr_timeEdit</tabstop>
  <tabstop>dep_comboBox</tabstop>
  <tabstop>arr_comboBox</tabstop>
  <tabstop>ac_comboBox</tabstop>
  <tabstop>pic_comboBox</tabstop>
  <tabstop>fr_comboBox</tabstop>
  <tabstop>fr_timeEdit</tabstop>
  <tabstop>day_spinBox</tabstop>
  <tabstop>nit_spinBox</tabstop>
  <tabstop>func_comboBox</tabstop>
  <tabstop>nit_checkBox</tabstop>
  <tabstop>remarks_textEdit</tabstop>
  <tabstop>sse_time</tabstop>
  <tabstop>sme_time</tabstop>
  <tabstop>mcc_timeEdit</tabstop>
  <tabstop>tot_timeEdit</tabstop>
  <tabstop>ifr_timeEdit</tabstop>
  <tabstop>nit_timeEdit</tabstop>
  <tabstop>pic_timeEdit</tabstop>
  <tabstop>dual_timeEdit</tabstop>
  <tabstop>copi_timeEdit</tabstop>
  <tabstop>instr_timeEdit</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
